packages/core/src/amazonqFeatureDev/client/featureDev.ts

```diff
@@ -26,8 +26,17 @@ import { createCodeWhispererChatStreamingClient } from '../../shared/clients/cod
 import { getClientId, getOptOutPreference, getOperatingSystem } from '../../shared/telemetry/util'
 import { extensionVersion } from '../../shared/vscode/env'
 
+// Re-enable once BE is able to handle retries.
+const writeAPIRetryOptions = {
```

Comment:
nit: naming, don't use camelcase for constants

Reply:
Variable name `WRITE_API_RETRY_OPTIONS` must match one of the following formats: camelCase, PascalCaseeslint[@typescript-eslint/naming-convention](https://typescript-eslint.io/rules/naming-convention)